### DISCUSSION TITLE
Vertical bar support

### DISCUSSION THREAD

**Discussion by Tnixc** [FelixKratz/SketchyBar#288] [Ideas]
Not only would this would be nice for ultrawide monitors, but it looks cool as hell.

![image](https://user-images.githubusercontent.com/85466117/201810989-7bee6470-40cf-472c-a4b6-77546d8a6701.jpeg)

https://reddit.com/r/unixporn/comments/yveint/hyprland_left_statusbar_is_a_huge_improvement/

**Comment by es183923** [comment]
Check #240
